www/src/libs/math.js

```diff
@@ -317,6 +317,26 @@ var _mod = {
        var x1=float_check(x);
        var y1=float_check(y);
        return float(Math.sqrt(x1*x1 + y1*y1))},
+    inf: float('inf'),
+    isclose:function(){
+        var $ns = $B.args("isclose",
+                          4,
+                          {a:null,b:null,rel_tol:null,abs_tol:null},
+                          ['a', 'b', 'rel_tol', 'abs_tol'],
+                          arguments,
+                          {rel_tol:1e-09, abs_tol:0.0}, 
+                          null, 
+                          null)
+        var a = $ns['a'];
+        var b = $ns['b'];
+        var rel_tol = $ns['rel_tol'];
+        var abs_tol = $ns['abs_tol'];
+        if (_b_.$isinf(a) || _b_.$isinf(b)){return false}
+        if (rel_tol < 0.0 || abs_tol < 0.0) throw ValueError('tolerances must be non-negative')
+        if (a == b){return True}
```

Comment:
This should go to before the `$isinf` test on line 334 (inf should be close to inf as well as -inf to -inf as per the PEP and IEEE rules)